src/ddmd/root/array.d

```diff
@@ -199,12 +199,12 @@ public:
 struct BitArray
 {
 nothrow:
-    size_t length() const pure
+    @property size_t length() const pure
```

Comment:
Hmmmm... this code should not be broken because it is not ambiguous. If you have `size_t length();` and `void length(size_t)` it is unambiguous in all contexts what the get and set mean.

Ambiguity occurs when a property has a default value and returns a ref.